portfolio/src/main/webapp/commentScript.js

```diff
@@ -32,7 +32,7 @@ var SPACE1_PARAMETER = '\n';
 var SPACE3_PARAMETER = '\n\n\n';
 
 function getComment(){
-    const responsePromise = fetch(COMMENT_SERVLET);
+    const responsePromise = fetch('/comment');
```

Comment:
Why remove the COMMENT_SERVLET?

Reply:
It was a hypothesis that i proposed when the javaservlet could not fetch the comment as a 500 fetch error so I changed it (This is fixed in the many_dev branch)